### PR TITLE
add models

### PR DESCRIPTION
**Pull request by jeffersonoliveira-dev**
Criei os modelos e fiz a conexão entre eles no db. 

fixes https://trello.com/c/5wtU0qma/1-modelar-banco

### CODE REVIEW

File: app/models/availability.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Availability < ActiveRecord::Base
+  validates :date1, :date2, :date3, :date4, presence: true
```
Comment (eliasdevelop) [inline app/models/availability.rb]: Troca pra date_1, date_2 etc

File: log/development.log [hunk 2]
```diff
@@ -1,3 +1,537 @@
...
+JOIN pg_class t1 ON c.conrelid = t1.oid
+JOIN pg_class t2 ON c.confrelid = t2.oid
+JOIN pg_attribute a1 ON a1.attnum = c.conkey[1] AND a1.attrelid = t1.oid
+JOIN pg_attribute a2 ON a2.attnum = c.confkey[1] AND a2.attrelid = t2.oid
+JOIN pg_namespace t3 ON c.connamespace = t3.oid
+WHERE c.contype = 'f'
+  AND t1.relname = 'students'
+  AND t3.nspname = ANY (current_schemas(false))
+ORDER BY c.conname
+
```
Comment (eliasousa) [inline log/development.log]: Adiciona esse cara no .gitignore e remove do github

File: test/fixtures/addresses.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  street: MyString
+  district: MyString
+  city: MyString
+  cep: MyString
+
+two:
+  street: MyString
+  district: MyString
+  city: MyString
+  cep: MyString
```
Comment (eliasousa) [inline test/fixtures/addresses.yml]: Remove as fixtures

File: test/models/address_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (eliasousa) [inline test/models/address_test.rb]: Remove esses tests tbm plz

File: log/development.log (hunk 2 above)
Comment (jeffersonoliveira-dev) [reply log/development.log]: fixed

File: test/fixtures/addresses.yml (hunk 3 above)
Comment (jeffersonoliveira-dev) [reply test/fixtures/addresses.yml]: removidas

File: test/models/address_test.rb (hunk 4 above)
Comment (jeffersonoliveira-dev) [reply test/models/address_test.rb]: done

**Comment by eliasousa** [comment]
:shipit:

**Pull request merged**